src/Data/Nat/Divisibility.agda

```diff
@@ -25,24 +25,68 @@ open import Relation.Binary.Definitions
   using (Reflexive; Transitive; Antisymmetric; Decidable)
 import Relation.Binary.Reasoning.Preorder as PreorderReasoning
 open import Relation.Binary.PropositionalEquality.Core
-  using (_≡_; _≢_; refl; sym; trans; cong; cong₂; subst)
+  using (_≡_; _≢_; refl; sym; cong; subst)
 open import Relation.Binary.Reasoning.Syntax
-import Relation.Binary.PropositionalEquality.Properties as PropEq
+open import Relation.Binary.PropositionalEquality.Properties
+  using (isEquivalence; module ≡-Reasoning)
+
+private
+  variable d m n o p : ℕ
+
+
+------------------------------------------------------------------------
+-- Definition and derived properties
+
+open import Data.Nat.Divisibility.Core public hiding (*-pres-∣)
+
+module _ (m∣n : m ∣ n) where
+
+  open _∣_ m∣n
+  open ≤-Reasoning
+
+  instance
+    quotient≢0 : .{{NonZero n}} → NonZero quotient
+    quotient≢0 rewrite equality = m*n≢0⇒m≢0 quotient
+
+  equalityᵒ : n ≡ m * quotient
+  equalityᵒ rewrite equality = *-comm quotient m
+
+  quotient-∣ : quotient ∣ n
+  quotient-∣ = divides m equalityᵒ
+
+  quotient>1 : m < n → 1 < quotient
+  quotient>1 m<n = *-cancelˡ-< m 1 quotient $ begin-strict
+      m * 1        ≡⟨ *-identityʳ m ⟩
+      m            <⟨ m<n ⟩
+      n            ≡⟨ equalityᵒ ⟩
+      m * quotient ∎
+
+  quotient-< : .{{NonTrivial m}} → .{{NonZero n}} → quotient < n
+  quotient-<     = begin-strict
+    quotient     <⟨ m<m*n quotient m (nonTrivial⇒n>1 m) ⟩
+    quotient * m ≡⟨ equality ⟨
+    n            ∎
+
+-- Exports
+
+open _∣_ using (quotient) public
```

Comment:
This export doesn't do anything as opening `Divisibility.Core` above already exports `quotient`

Reply:
Ah, but see elsewhere. It doesn't now...